velero-plugin-for-microsoft-azure/object_store.go

```diff
@@ -221,7 +235,11 @@ func (o *ObjectStore) Init(config map[string]string) error {
 		return err
 	}
 
-	// 6. get storageClient and blobClient
+	// 1. get storageClient and blobClient
+	if _, err := getRequiredValues(mapLookup(config), storageAccountConfigKey); err != nil {
```

Comment:
I think we want to remove this here too, since the static storage account key is not required and will be fetched properly (either from secret or from API) via `getStorageAccountKey(...)` call above

Reply:
Also here the `config[storageAccountConfigKey]` is used a few lines after, see velero-plugin-for-microsoft-azure/object_store.go:241. Previously the check was done in `getStorageAccountKey` function, but now that branch could be skipped when the new feature is used.

Reply:
Yep, you're right, thanks.